main.js

```diff
@@ -106,25 +110,31 @@ function changeBackgroundImg() {
   }
 }
 
-clearInputField(saveOutfit);
-
 function clearInputField(saveOutfit) {
   outFitInput.value = "";
-  // dressBear();
-  // saveOutfit.classList.add('disable-save')
   disableSaveButton(event);
 }
 
 function disableSaveButton(event){
   if(outFitInput.value != "") {
-    console.log('yo');
-    saveOutfit.classList.remove('disable-save');
-  } else(saveOutfit.classList.add('disable-save'))
+    saveOutfitBtn.classList.remove('disable-save');
+    saveOutfitBtn.removeAttribute('disabled');
+  } else {
+    saveOutfitBtn.classList.add('disable-save')
+    saveOutfitBtn.setAttribute('disabled', '');
+  }
```

Comment:
I added code to change the disabled attribute of the input so it doesn't create card without any text